WordPress/Classes/ViewRelated/Blog/SharingConnectionsViewController.m

```diff
@@ -231,6 +231,15 @@ - (void)handleConnectTapped:(NSIndexPath *)indexPath
     [self.helper connectPublicizeService];
 }
 
+- (void)handleLearnMoreTappedWithLearnMoreURL:(NSURL*)learnMoreURL
```

Comment:
I think it makes sense that you stuck with `ValidationError.continueURL` instead of my suggested `ValidationError.learnMoreURL`. However, I think we should be consistent. Since you stuck with `continueURL`, we should call this:

```suggestion
- (void)handleContinueURLTapped:(NSURL*)url
```